### PR TITLE
Add `context` to arguments for `execute` function #195

### PR DESCRIPTION
**Pull request by anatol-sialitski**

### CODE REVIEW

File: docs/advanced.adoc [hunk 1]
```diff
@@ -184,6 +184,10 @@ List of properties for `params` object:
 |schemaOptions:Object
 |SchemaOptions object to customize schema. Property is optional.
 |
+
+|queryContext:Object
```
Comment (rymsha) [inline docs/advanced.adoc:188]: Why not `context`?

File: docs/advanced.adoc [hunk 2]
```diff
@@ -184,6 +184,10 @@ List of properties for `params` object:
...
+|JS Object (Optional) It allows to share data between fields. To read values from it in a resolve function use `env.context`.
```
Comment (rymsha) [inline docs/advanced.adoc:189]: Feels like a different  writing style if I compare with other parameter descriptions

**Review by rymsha — CHANGES_REQUESTED** [review]
It looks like a good idea, but why
- it was not implemented before?
- it is required now?

**Review by rymsha — APPROVED** [review]

**Review by alansemenov — APPROVED** [review]
In general, use `const` (instead of `let`) for variables that won't change their values.

**Pull request merged**